torch/csrc/jit/passes/shape_analysis.cpp

```diff
@@ -787,15 +839,9 @@ class ShapePropagator {
     static const register_formula_for simple_unary_ops{
         {
             "aten::abs(Tensor self) -> Tensor",
-            "aten::acos(Tensor self) -> Tensor",
```

Comment:
Unary Floating Ufuncs have been moved from this list to the new list (unary_floating_ufuncs).